bolt-sidecar/src/client/commit_boost.rs

```diff
@@ -39,8 +39,8 @@ pub enum CommitBoostError {
 #[allow(unused)]
 impl CommitBoostSigner {
     /// Create a new [CommitBoostSigner] instance
-    pub async fn new(signer_server_address: String, jwt: &str) -> Result<Self, CommitBoostError> {
-        let signer_client = SignerClient::new(signer_server_address, jwt)?;
+    pub async fn new(signer_server_address: &str, jwt: &str) -> Result<Self, CommitBoostError> {
```

Comment:
Nit but usually the function interface should show if it requires an owned or borrowed variable, i.e. it shouldn't do a hidden clone